react/data/schema.graphql

```diff
@@ -1,21 +1,37 @@
-# PR #1680 27ab15e5297862f0bf2717b6b0b98d99eb609e26
-
 schema {
```

Comment:
Please add the PR number and commit hash to the first line.

Reply:
Except for this, everything else is LGTM.

Reply:
Done! Thanks :)